angucomplete-alt.js

```diff
@@ -298,20 +298,35 @@
         }
       }
 
+      function handleSubmitOnEnter(event) {
+        if (event && !scope.submitOnEnter) {
```

Comment:
Yep, I'm seeing the updated code. It sets it to the result on load, but it seems the handler uses the original value.